chatbot.py

```diff
@@ -13,25 +13,33 @@
 from googleMapsApiModule import get_from_maps_database
 from googleMapsApiModule import geocoding
 
+import enum
+from enum import Enum, auto
+class LearnResponse(enum.Enum): 
+    message = auto()
+    train_me = auto()
+    origin = auto()
+    destination = auto()
```

Comment:
Just a minor nitpicking: these constant enum keys (and constants in general) are usually denoted in all capitals. Like `MESSAGE` over `message`. 😄

Reply:
okay sir, I' ll change these.